lib/urlwatch/reporters.py

```diff
@@ -407,9 +409,10 @@ def submit(self):
         if self.config['html']:
             body_html = '\n'.join(self.convert(HtmlReporter).submit())
 
-            msg = mailer.msg_html(self.config['from'], self.config['to'], subject, body_text, body_html)
+            msg = mailer.msg_html(self.config['from'], self.config['to'], subject[:subject_length],
```

Comment:
You cut off the subject in two places now, why not do it above in a single place where the `subject` is first assigned?